src/collection/algorithms/dijkstra.js

```diff
@@ -74,7 +74,7 @@ var elesfn = ({
 
       knownDist[ uid ] = smalletsDist;
 
-      if( smalletsDist === Math.Infinite ){
+      if( smalletsDist === undefined ){
```

Comment:
This is a good catch, but it should be checking `Infinity` rather than `Math.Infinite`.  Looks like a typo